jerry-core/config.h

```diff
@@ -39,6 +39,7 @@
 # define CONFIG_DISABLE_ES2015_ARROW_FUNCTION
 # define CONFIG_DISABLE_ES2015_CLASS
 # define CONFIG_DISABLE_ES2015_BUILTIN
+# define CONFIG_DISABLE_ES2015_OBJECT_LITERAL
```

Comment:
It is a new feature, so please update the [profile](https://github.com/jerryscript-project/jerryscript/blob/master/jerry-core/profiles/README.md) documentation.